code/datums/uplink/uplink_items.dm

```diff
@@ -132,7 +131,9 @@ datum/uplink_item/dd_SortValue()
 	return I
 
 /datum/uplink_item/item/get_goods(var/obj/item/device/uplink/U, var/loc)
-	var/obj/item/I = new path(loc)
+	var/list/input_args = new_args.Copy()
+	input_args.Insert(1, loc)
```

Comment:
No, it's fine. The 1 isn't the argument given to new, it's saying make the 1st element of input_args into the first argument and then shift the other ones up a place to ensure that the loc is inserted at position one.